Describe license key errors (#5657)
DevExpress/devextreme-documentation: api-reference/50 Common/utils/zz Errors and Warnings/W0020.md

---
---
##### shortDescription
Occurs when the license key is expired.

---

> A mismatch exists between license key and DevExtreme version.

Reviewer: ```suggestion
The license key does not match the DevExtreme version.
```

To proceed, you can:

> - use a version of DevExtreme linked to your license key (see [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/))

Reviewer: ```suggestion
- use the DevExtreme version linked to your license key (see [Download Manager](https://www.devexpress.com/ClientCenter/DownloadManager/))
```

> - [renew your DevExpress Subscription](https://www.devexpress.com/buy/renew/) (once you renew your subscription, you will be entitled to product updates and support service)

Reviewer: ```suggestion
- [renew your DevExpress Subscription](https://www.devexpress.com/buy/renew/) (once you renew your subscription, you can get product updates and support)
```

> If you have licensing-related questions or need help with a renewal, please email `clientservices@devexpress.com`. We will be happy to follow-up.

Reviewer: ```suggestion
If you have licensing-related questions or need help with a renewal, please email `clientservices@devexpress.com`. We will be happy to assist you.
```
